### PR TITLE
fix: remove indirect dependency on Google services from F-Droid build profile

### PR DESCRIPTION
**Pull request by sysvinit**
----
#### PR Submission Checklist for internal contributors

- The **PR Title**
  - [x] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
  - [ ] contains a reference JIRA issue number like `SQPIT-764`
  - [x] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
  - [x] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

### Issues

F-Droid removed `wire-android` from the F-Droid store, due to a linter discovering that the APK files contained Google services code -- see [this comment](https://gitlab.com/fdroid/fdroidserver/-/issues/1004#note_1028069460) on F-Droid's GitLab repo.

### Causes

The app depends on https://github.com/evernote/android-job for handling some background processing tasks (such as handling push notifications). In order to provide compatibility with a range of Android versions, `android-job` has a number of different backend implementations for handling scheduling and execution of background jobs.

One of these backends use Google Play services as a fallback for Android versions which don't provide any other suitable functionality which the library can piggy-back onto. In spite of the fact that the F-Droid build profile does not include any *direct* dependencies on closed Google code, `android-job`'s own compile-time dependency causes some GMS-related code to leak into the build artefacts.

### Solutions

Recent versions of Android provide equivalent functionality to `android-job` as part of the runtime/SDK, known as `WorkManager`. While some parts of the app do seem to use `WorkManager` already, due to lack of resources we've decided not to try porting the code still using `android-job`.

Instead, we've forked the `android-job` library to https://github.com/wireapp/android-job, and I've pushed a feature branch (based on the version of which `wire-android` depends on) which has all references to proprietary Google code removed.

In the mean time, @julialongtin has been in contact with the F-Droid maintainers, and they report that quite a few apps (especially older ones) have been removed in the same linting pass due to depending on `android-job`. However, they aren't aware of anyone else who's actively working on removing the Google dependencies from that library, and if we were to publish our fork (presumably in the sense of providing build artefacts in a repository like Maven or Sonatype), then they would be able to use it to monkey-patch the other apps affected by the linting check.

While we would like to be able to release a fixed version of `android-job`, arranging this will take some time (especially if we need to port our changes to several versions of the library), and we want to fix `wire-android` quickly so it can be made available in F-Droid again as soon as possible.

This PR contains a number of changes to this end. I've used `git-subtree(1)` to import our patched copy of `android-job` into the repository, and set up some glue in Gradle so that the F-Droid build profile uses the patched version in the repository, while Play Store builds should continue to use the upstream version. I have **specifically avoided using submodules** here, because once submodules are added to a git repository, they're *really* annoying to deal with if they ever get removed. If/when we publish a fork of `android-job`, this leaves a way to cleanly get rid of the locally imported code and pull down a prebuilt copy of the library with Gradle instead.

### Testing

F-Droid spec clients built from this PR branch no longer trigger the error from F-Droid's linting tool:

```
(hacks) molly on flux (1) ~/s/w/wire-android> fdroid scanner --verbose --exodus wire-fdroid-bad.apk
2022-07-21 15:11:27,683 WARNING: apksigner not found! Cannot sign or verify modern APKs
2022-07-21 15:11:27,685 DEBUG: Starting new HTTPS connection (1): reports.exodus-privacy.eu.org:443
2022-07-21 15:11:28,174 DEBUG: https://reports.exodus-privacy.eu.org:443 "GET /api/trackers HTTP/1.1" 200 None
2022-07-21 15:11:28,312 DEBUG: 428 trackers signatures loaded
2022-07-21 15:11:28,332 INFO: Scanning APK with dexdump for known non-free classes.
2022-07-21 15:11:28,386 DEBUG: > /usr/bin/dexdump /tmp/tmpxgxfm_eg/classes.dex
2022-07-21 15:11:33,825 DEBUG: > /usr/bin/dexdump /tmp/tmpxgxfm_eg/classes2.dex
2022-07-21 15:11:39,397 DEBUG: > /usr/bin/dexdump /tmp/tmpxgxfm_eg/classes3.dex
2022-07-21 15:11:39,693 DEBUG: Found class 'com/google/android/gms/gcm/GcmNetworkManager'
2022-07-21 15:11:39,694 DEBUG: Found class 'com/google/android/gms/gcm/Task'
2022-07-21 15:11:39,710 DEBUG: Found class 'com/google/android/gms/gcm/GcmTaskService'
2022-07-21 15:11:40,350 CRITICAL: Found problems in wire-fdroid-bad.apk
2022-07-21 15:11:40,352 WARNING: Scanner found 3 problems in wire-fdroid-bad.apk:
(hacks) molly on flux (1) ~/s/w/wire-android> fdroid scanner --verbose --exodus wire-fdroid-fixed.apk
2022-07-21 15:11:44,151 WARNING: apksigner not found! Cannot sign or verify modern APKs
2022-07-21 15:11:44,153 DEBUG: Starting new HTTPS connection (1): reports.exodus-privacy.eu.org:443
2022-07-21 15:11:44,624 DEBUG: https://reports.exodus-privacy.eu.org:443 "GET /api/trackers HTTP/1.1" 200 None
2022-07-21 15:11:44,753 DEBUG: 428 trackers signatures loaded
2022-07-21 15:11:44,772 INFO: Scanning APK with dexdump for known non-free classes.
2022-07-21 15:11:44,826 DEBUG: > /usr/bin/dexdump /tmp/tmpgdhkmg2k/classes.dex
2022-07-21 15:11:50,190 DEBUG: > /usr/bin/dexdump /tmp/tmpgdhkmg2k/classes2.dex
2022-07-21 15:11:55,799 DEBUG: > /usr/bin/dexdump /tmp/tmpgdhkmg2k/classes3.dex
```

### CODE REVIEW

**Comment by AndroidBob** [comment]
Build Success
ID:1
URL:[Link to Buildjob](https://10.10.124.134/job/wire-android/job/PR-3719/1/)

**Comment by sysvinit** [comment]
Running the linter on the CI artefacts comes back clean:

<details>
<summary> (Click to expand) </summary>
<pre>
(hacks) molly on flux (1) ~/dl> fdroid scanner --verbose --exodus wire-fdroid-debug-3.81.1.apk 
2022-07-21 17:00:30,166 WARNING: apksigner not found! Cannot sign or verify modern APKs
2022-07-21 17:00:30,167 DEBUG: Starting new HTTPS connection (1): reports.exodus-privacy.eu.org:443
2022-07-21 17:00:30,616 DEBUG: https://reports.exodus-privacy.eu.org:443 "GET /api/trackers HTTP/1.1" 200 None
2022-07-21 17:00:30,749 DEBUG: 428 trackers signatures loaded
2022-07-21 17:00:30,768 INFO: Scanning APK with dexdump for known non-free classes.
2022-07-21 17:00:30,783 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes.dex
2022-07-21 17:00:31,099 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes10.dex
2022-07-21 17:00:36,736 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes100.dex
2022-07-21 17:00:36,949 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes101.dex
2022-07-21 17:00:37,209 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes102.dex
2022-07-21 17:00:37,423 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes103.dex
2022-07-21 17:00:37,681 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes104.dex
2022-07-21 17:00:37,948 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes105.dex
2022-07-21 17:00:38,159 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes106.dex
2022-07-21 17:00:38,369 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes107.dex
2022-07-21 17:00:38,580 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes108.dex
2022-07-21 17:00:38,789 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes109.dex
2022-07-21 17:00:39,001 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes11.dex
2022-07-21 17:00:39,254 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes110.dex
2022-07-21 17:00:39,467 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes111.dex
2022-07-21 17:00:39,677 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes112.dex
2022-07-21 17:00:39,955 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes113.dex
2022-07-21 17:00:40,166 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes114.dex
2022-07-21 17:00:40,378 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes115.dex
2022-07-21 17:00:40,587 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes116.dex
2022-07-21 17:00:40,801 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes117.dex
2022-07-21 17:00:41,057 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes118.dex
2022-07-21 17:00:41,324 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes119.dex
2022-07-21 17:00:41,583 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes12.dex
2022-07-21 17:00:41,792 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes120.dex
2022-07-21 17:00:42,014 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes121.dex
2022-07-21 17:00:42,275 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes122.dex
2022-07-21 17:00:42,486 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes123.dex
2022-07-21 17:00:42,748 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes124.dex
2022-07-21 17:00:43,014 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes125.dex
2022-07-21 17:00:43,224 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes126.dex
2022-07-21 17:00:43,475 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes127.dex
2022-07-21 17:00:43,731 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes128.dex
2022-07-21 17:00:43,941 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes129.dex
2022-07-21 17:00:44,165 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes13.dex
2022-07-21 17:00:44,376 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes130.dex
2022-07-21 17:00:44,589 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes131.dex
2022-07-21 17:00:44,800 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes132.dex
2022-07-21 17:00:45,008 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes133.dex
2022-07-21 17:00:45,218 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes134.dex
2022-07-21 17:00:45,429 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes135.dex
2022-07-21 17:00:45,640 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes136.dex
2022-07-21 17:00:45,852 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes137.dex
2022-07-21 17:00:46,062 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes138.dex
2022-07-21 17:00:46,273 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes139.dex
2022-07-21 17:00:46,482 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes14.dex
2022-07-21 17:00:46,694 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes140.dex
2022-07-21 17:00:46,970 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes141.dex
2022-07-21 17:00:47,179 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes142.dex
2022-07-21 17:00:47,397 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes143.dex
2022-07-21 17:00:47,657 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes144.dex
2022-07-21 17:00:47,873 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes145.dex
2022-07-21 17:00:48,160 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes146.dex
2022-07-21 17:00:48,370 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes147.dex
2022-07-21 17:00:48,582 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes148.dex
2022-07-21 17:00:48,800 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes149.dex
2022-07-21 17:00:49,055 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes15.dex
2022-07-21 17:00:49,311 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes150.dex
2022-07-21 17:00:49,522 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes151.dex
2022-07-21 17:00:49,785 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes152.dex
2022-07-21 17:00:50,085 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes153.dex
2022-07-21 17:00:55,204 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes154.dex
2022-07-21 17:00:55,420 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes155.dex
2022-07-21 17:00:55,681 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes156.dex
2022-07-21 17:00:55,901 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes157.dex
2022-07-21 17:00:56,115 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes158.dex
2022-07-21 17:00:56,327 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes159.dex
2022-07-21 17:00:56,540 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes16.dex
2022-07-21 17:00:56,752 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes160.dex
2022-07-21 17:00:56,965 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes161.dex
2022-07-21 17:00:57,175 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes162.dex
2022-07-21 17:00:57,385 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes163.dex
2022-07-21 17:00:57,598 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes164.dex
2022-07-21 17:00:57,808 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes165.dex
2022-07-21 17:00:58,065 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes166.dex
2022-07-21 17:00:58,280 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes167.dex
2022-07-21 17:00:58,540 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes168.dex
2022-07-21 17:00:58,754 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes169.dex
2022-07-21 17:00:59,013 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes17.dex
2022-07-21 17:00:59,225 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes170.dex
2022-07-21 17:00:59,475 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes171.dex
2022-07-21 17:00:59,688 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes172.dex
2022-07-21 17:00:59,944 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes173.dex
2022-07-21 17:01:00,153 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes174.dex
2022-07-21 17:01:00,365 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes175.dex
2022-07-21 17:01:00,576 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes176.dex
2022-07-21 17:01:00,829 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes177.dex
2022-07-21 17:01:01,039 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes178.dex
2022-07-21 17:01:01,255 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes179.dex
2022-07-21 17:01:01,469 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes18.dex
2022-07-21 17:01:01,738 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes180.dex
2022-07-21 17:01:02,338 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes181.dex
2022-07-21 17:01:06,476 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes182.dex
2022-07-21 17:01:11,590 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes183.dex
2022-07-21 17:01:15,789 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes184.dex
2022-07-21 17:01:19,781 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes19.dex
2022-07-21 17:01:20,044 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes2.dex
2022-07-21 17:01:20,309 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes20.dex
2022-07-21 17:01:20,520 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes21.dex
2022-07-21 17:01:20,740 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes22.dex
2022-07-21 17:01:20,956 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes23.dex
2022-07-21 17:01:21,215 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes24.dex
2022-07-21 17:01:21,459 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes25.dex
2022-07-21 17:01:21,674 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes26.dex
2022-07-21 17:01:21,929 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes27.dex
2022-07-21 17:01:22,197 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes28.dex
2022-07-21 17:01:22,592 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes29.dex
2022-07-21 17:01:22,847 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes3.dex
2022-07-21 17:01:23,106 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes30.dex
2022-07-21 17:01:23,321 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes31.dex
2022-07-21 17:01:23,535 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes32.dex
2022-07-21 17:01:23,805 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes33.dex
2022-07-21 17:01:24,069 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes34.dex
2022-07-21 17:01:24,280 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes35.dex
2022-07-21 17:01:24,491 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes36.dex
2022-07-21 17:01:24,706 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes37.dex
2022-07-21 17:01:24,967 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes38.dex
2022-07-21 17:01:25,182 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes39.dex
2022-07-21 17:01:25,450 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes4.dex
2022-07-21 17:01:25,698 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes40.dex
2022-07-21 17:01:25,956 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes41.dex
2022-07-21 17:01:26,168 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes42.dex
2022-07-21 17:01:26,436 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes43.dex
2022-07-21 17:01:26,648 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes44.dex
2022-07-21 17:01:26,862 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes45.dex
2022-07-21 17:01:27,075 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes46.dex
2022-07-21 17:01:27,289 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes47.dex
2022-07-21 17:01:27,504 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes48.dex
2022-07-21 17:01:27,716 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes49.dex
2022-07-21 17:01:27,928 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes5.dex
2022-07-21 17:01:28,142 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes50.dex
2022-07-21 17:01:28,355 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes51.dex
2022-07-21 17:01:28,603 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes52.dex
2022-07-21 17:01:28,874 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes53.dex
2022-07-21 17:01:29,087 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes54.dex
2022-07-21 17:01:29,340 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes55.dex
2022-07-21 17:01:29,596 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes56.dex
2022-07-21 17:01:29,853 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes57.dex
2022-07-21 17:01:30,119 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes58.dex
2022-07-21 17:01:30,338 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes59.dex
2022-07-21 17:01:30,593 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes6.dex
2022-07-21 17:01:30,848 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes60.dex
2022-07-21 17:01:31,061 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes61.dex
2022-07-21 17:01:31,327 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes62.dex
2022-07-21 17:01:31,541 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes63.dex
2022-07-21 17:01:31,759 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes64.dex
2022-07-21 17:01:32,026 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes65.dex
2022-07-21 17:01:32,235 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes66.dex
2022-07-21 17:01:32,448 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes67.dex
2022-07-21 17:01:32,661 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes68.dex
2022-07-21 17:01:32,882 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes69.dex
2022-07-21 17:01:33,098 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes7.dex
2022-07-21 17:01:33,314 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes70.dex
2022-07-21 17:01:33,578 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes71.dex
2022-07-21 17:01:33,842 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes72.dex
2022-07-21 17:01:34,060 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes73.dex
2022-07-21 17:01:34,312 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes74.dex
2022-07-21 17:01:34,522 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes75.dex
2022-07-21 17:01:34,738 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes76.dex
2022-07-21 17:01:35,002 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes77.dex
2022-07-21 17:01:35,218 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes78.dex
2022-07-21 17:01:35,435 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes79.dex
2022-07-21 17:01:35,647 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes8.dex
2022-07-21 17:01:35,866 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes80.dex
2022-07-21 17:01:36,123 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes81.dex
2022-07-21 17:01:36,337 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes82.dex
2022-07-21 17:01:36,552 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes83.dex
2022-07-21 17:01:36,817 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes84.dex
2022-07-21 17:01:37,032 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes85.dex
2022-07-21 17:01:37,292 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes86.dex
2022-07-21 17:01:37,566 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes87.dex
2022-07-21 17:01:37,778 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes88.dex
2022-07-21 17:01:37,997 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes89.dex
2022-07-21 17:01:38,256 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes9.dex
2022-07-21 17:01:38,468 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes90.dex
2022-07-21 17:01:38,681 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes91.dex
2022-07-21 17:01:38,894 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes92.dex
2022-07-21 17:01:39,111 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes93.dex
2022-07-21 17:01:39,368 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes94.dex
2022-07-21 17:01:39,582 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes95.dex
2022-07-21 17:01:39,794 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes96.dex
2022-07-21 17:01:40,007 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes97.dex
2022-07-21 17:01:40,228 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes98.dex
2022-07-21 17:01:40,441 DEBUG: > /usr/bin/dexdump /tmp/tmpd07x3mlc/classes99.dex
</pre>
</details>

**Comment by vitorhugods** [comment]
🎉 Nice and clean.
I was going to suggest a git submodule but you have a great point there, really well thought.

File: app/build.gradle [hunk 1]
```diff
@@ -419,7 +419,12 @@ dependencies {
     implementation BuildDependencies.kotlinXSerialization
 
     //third party libraries
-    implementation BuildDependencies.androidJob
+    if (fdroidBuild) {
```
Comment (marcoconti83) [inline app/build.gradle:422]: I like the `if`, you have been very cautious by changing this only where it is strictly required to change 👍 

Perhaps we can promote the modified version to non-F-Droid builds too after seeing how it behaves on F-Droid for a bit. Although probably there's not enough time left to justify further polishing before this codebase reaches end-of-life.

**Review by marcoconti83 — APPROVED** [review]

**Pull request merged**